src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -425,4 +432,18 @@ public String inviteUser(String role) {
         activeUserId = (String) pumInternalUserResponse.get("userIdentifier");
         return bearerToken;
     }
+
+    @Test
+    @ToggleEnable(mapKey = "OrganisationMfaStatusController.retrieveMfaStatusByUserId", withFeature = true)
+    public void findMFAScenario() {
+        setUpOrgTestData();
```

Comment:
Does this method need to be called again if it is called on line 62?

Reply:
yes, because even though it is part of the same class, this is considered as a separate test as we have added the @Test annotation, so it needs to be run again.

Reply:
Can it not be ran in the same test method as the other classes? Otherwise we will be creating extra organisations and users